workflows/sock-shop-demo/usingCMDProbe/workflow.yaml

```diff
@@ -488,6 +560,85 @@ spec:
     
+    - name: pod-network-loss
+      inputs:
+        artifacts:
+          - name: pod-network-loss
+            path: /tmp/chaosengine.yaml
+            raw:
+              data: |
+                apiVersion: litmuschaos.io/v1alpha1
+                kind: ChaosEngine
+                metadata:
+                  name: pod-network-loss-chaos
+                  namespace: {{workflow.parameters.adminModeNamespace}}
+                spec:
+                  appinfo:
+                    appns: 'sock-shop'
+                    applabel: 'name=user-db'
+                    appkind: 'statefulset'
+                  jobCleanUpPolicy: retain
+                  monitoring: true
+                  annotationCheck: 'false'
+                  engineState: 'active'
+                  auxiliaryAppInfo: ''
+                  chaosServiceAccount: litmus-admin
+                  components:
+                    runner:
+                      imagePullPolicy: Always
+                  experiments:
+                    - name: pod-network-loss
+                      spec:
+                        probe:
+                        - name: "check-frontend-access-url"
+                          type: "httpProbe"
+                          httpProbe/inputs:
+                            url: "http://front-end.sock-shop.svc.cluster.local:80/cards"
+                            expectedResponseCode: "200"
+                          mode: "Continuous"
+                          runProperties:
+                            probeTimeout: 2
+                            interval: 1
+                            retry: 1
+                            probePollingInterval: 1
+                        - name: "check-benchmark"
+                          type: "cmdProbe"
+                          cmdProbe/inputs:
+                            command: "curl http://qps-test.sock-shop.svc.cluster.local"
+                            comparator:
+                              type: "int" # supports: string, int, float
+                              criteria: ">=" #supports >=,<=,>,<,==,!= for int and contains,equal,notEqual,matches,notMatches for string values
+                              value: "500"
+                            source: "inline" # it can be “inline” or any image
+                          mode: "Edge"
+                          runProperties:
+                            probeTimeout: 2
+                            interval: 2
+                            retry: 1
+                            initialDelaySeconds: 2
+                        components:
+                          env:
+                            - name: TARGET_CONTAINER
+                              value: ''
+                            - name: LIB_IMAGE
+                              value: 'litmuschaos/go-runner:latest'
+                            - name: TOTAL_CHAOS_DURATION
+                              value: '30' 
+                            
```

Comment:
uniform the newline b/w envs (0/1)

based on the above comments do the same changes in other workflows as well